scripts/zones/The_Colosseum/npcs/Zandjarl.lua

```diff
@@ -0,0 +1,60 @@
+-----------------------------------
+--  Area: The Colosseum
+--  NPC:  Zandjarl
+--  Type: Pankration NPC
+-----------------------------------
+package.loaded["scripts/zones/The_Colosseum/TextIDs"] = nil;
```

Comment:
You can literally copy paste the file at the same link I sent you yesterday. It's been updated since I tested the param you said wasn't working, which turned out to in fact work.